lms/static/coffee/spec/instructor_dashboard/send_email_spec.coffee

```diff
@@ -0,0 +1,82 @@
+describe "Bulk Email Queueing", ->
+    beforeEach ->
+        testSubject = "Test Subject"
+        testBody =  "Hello, World! This is a test email message!"
+        loadFixtures 'coffee/fixtures/send_email.html'
+        @send_email = new SendEmail $('.send-email')
+        @send_email.$subject.val(testSubject)
+        @send_email.$send_to.first().prop("checked", true)
+        @send_email.$emailEditor =
+            save: ->
+                {"data": testBody}
+        @ajax_params = {
+            type: "POST",
+            dataType: "json",
+            url: undefined,
+            data: {
+                action: "send",
+                send_to: JSON.stringify([@send_email.$send_to.first().val()]),
+                subject: testSubject,
+                message: testBody,
+            },
+            success: jasmine.any(Function),
+            error: jasmine.any(Function),
+        }
+
+    it 'binds the send email button to a click event', ->
```

Comment:
Given that the behavior after "click" is done is tested below, there is no need to separately verify that a click handler is registered.